### ISSUE TITLE
`AttributeError: module 'ax' has no attribute '__version__'`

### ISSUE THREAD

**Issue by sgbaird** [facebook/Ax#1438]
Maybe worth considering implementing `__version__`
```python
import ax
ax.__version__
```

> ```python
> AttributeError: module 'ax' has no attribute '__version__'
> ```

As an easy workaround, `pip show ax-platform`:
```bash
(matsci-opt-benchmarks) [<>@notchpeak1:matsci-opt-benchmarks]$ pip show ax-platform
Name: ax-platform
Version: 0.2.10
Summary: Adaptive Experimentation
Home-page: https://github.com/facebook/Ax
Author: Facebook, Inc.
Author-email: 
License: MIT
Location: /uufs/chpc.utah.edu/common/home/sparks-thermoelectric/software/pkg/miniconda3/envs/matsci-opt-benchmarks/lib/python3.10/site-packages
Requires: botorch, ipywidgets, jinja2, pandas, plotly, scikit-learn, scipy, typeguard
Required-by:
```

**Comment by mpolson64** [comment]
Good catch, we will add one next time we do a version release :)

**Comment by mpolson64** [comment]
Addressed in v0.3.0
